### PR TITLE
Santa Fe HEV 2022: add missing FW versions

### PR DESCRIPTION
**Pull request by rchybicki**

### CODE REVIEW

**Comment by sshane** [comment]
Can you provide the dongle id these came from?

**Comment by rchybicki** [comment]
> Can you provide the dongle id these came from?

of course - 36921882a3bf0f19

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -766,19 +766,26 @@ class Buttons:
   },
   CAR.SANTA_FE_HEV_2022: {
     (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ',
       b'\xf1\x8799110CL500\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:770]:
```suggestion
      b'\xf1\x00TMhe SCC FHCUP      1.00 1.00 99110-CL500         ',
```

File: selfdrive/car/hyundai/values.py [hunk 2]
```diff
@@ -766,19 +766,26 @@ class Buttons:
...
     ],
     (Ecu.eps, 0x7d4, None): [
       b'\xf1\x00TM  MDPS C 1.00 1.02 56310-CLAC0 4TSHC102',
+      b'\xf1\x00TM  MDPS R 1.00 1.05 57700-CL000 4TSHP105',
+      b'\xf1\x8757700-CL000\xf1\x00TM  MDPS R 1.00 1.05 57700-CL000 4TSHP105',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:775]:
```suggestion
```

File: selfdrive/car/hyundai/values.py [hunk 3]
```diff
@@ -766,19 +766,26 @@ class Buttons:
...
     ],
     (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00TMH MFC  AT EUR LHD 1.00 1.06 99211-S1500 220727',
       b'\xf1\x00TMH MFC  AT USA LHD 1.00 1.03 99211-S1500 210224',
     ],
     (Ecu.transmission, 0x7e1, None): [
+      b'\xf1\x00PSBG2333  E16\x00\x00\x00\x00\x00\x00\x00TTM2H16UA3I\x94\xac\x8f',
+      b'\xf1\x8795441-3D110\x00\xf1\x81E16\x00\x00\x00\x00\x00\x00\x00\xf1\x00PSBG2333  E16\x00\x00\x00\x00\x00\x00\x00TTM2H16UA3I\x94\xac\x8f',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:783]:
```suggestion
```

**Comment by sshane** [comment]
I just removed some duplicate versions from the other FW queries, looks good!

**Pull request merged**